src/mame/includes/dragon.h

```diff
@@ -69,6 +69,7 @@ class dragon64_state : public dragon_state
 	void dragon64(machine_config &config);
 	void tanodr64h(machine_config &config);
 	void dragon64h(machine_config &config);
+	DECLARE_WRITE_LINE_MEMBER( acia_irq );
```

Comment:
Putting aside the fact that this trampoline is currently broken and probably unnecessary, internal trampolines shouldn’t be public.